### ISSUE TITLE
Creating USB qube locks out usb mouse and keyboard 4.0-rc1

### ISSUE THREAD

**Issue by JPL1** [QubesOS/qubes-issues#2959]
#### Qubes OS version 
R4.0-rc1
#### Affected TemplateVMs (e.g., `fedora-23`, if applicable):
fedora-25

### Expected behavior:
Creation of a USB VM to handle plug in USB devices
### Actual behavior:
Starting USB VM crashed system. Cannot restart
### Steps to reproduce the behavior:
Create USB VM following this process https://www.qubes-os.org/doc/usb/ (steps 1 and 2)
Start USB VM - system will freeze usb mouse and keyboard, unresponsive
On restarting get various cannot find ramdisk errors. Cannot restart.
### General notes:
I am running Qubes from a USB stick so I guess this is the problem, but Dom0 does not register additional plugged in USB sticks. This is good for security but how do I mount them to an AppVM if I can't create a USB VM?

**Comment by marmarek** [comment]
USB VM is incompatible with booting from USB  - that's why it is grayed out during installation.
It is possible to get it working, if you have multiple USB controllers, but require manual device assignment (to leave USB controller used for booting in dom0).

**Comment by JPL1** [comment]
I'm not sure if something has changed between releases, or if I slected the wrong option on installation. When I reinstall R4.0 what option should I select so that USB devices are able to plug straight into Dom0 (as they do with my R3.2 USB installation now)?

**Comment by marmarek** [comment]
At first system boot there is a configuration wizard - make sure "Create USB qube for holding all US controllers (sys-usb)" is not selected. If you boot from USB, it should be disabled by default.

**Comment by JPL1** [comment]
I have reinstalled rc1 on the USB stick. 'Create USB qube' was disabled in the boot wizard. I selected all defaults. 
Plugging in a USB stick into one a USB port there is no pop-up message like with R3.2. 
However, it is detected by dom0:  
`qvm-block` identifies it as dom0:sdc 
And `qvm-block a 'work' dom0:sdc` mounts it to the work AppVM
But if I click on the (very ugly) USB icon (top right of tool bar) dom0:sdc does not show up.  

So, it seems the toolbar app does not recognise USB devices that are plugged in after booting.

**Comment by marmarek** [comment]
> So, it seems the toolbar app does not recognise USB devices that are plugged in after booting.

Yes, this will be fixed in next update (QubesOS/qubes-desktop-linux-manager#7)

**Comment by JPL1** [comment]
Good to hear :) Will there still be a pop-up message when you plug in a USB? I find it useful

**Comment by ben-grande** [comment]
Can't be solved with one USB controller, at least without creating multiple USB qubes if there are enough controllers. There is another issue for this... I just can't find now.
